docs-chef-io/content/inspec/cli.md

```diff
@@ -64,7 +64,7 @@ inspec check PATH
 This subcommand has additional options:
 
 * ``--format=FORMAT``
-
+    The output format to use doc (default), json. If valid format is not provided then it will use the default.
```

Comment:
Is this just checking that the inspec.yml is in a particular format? Wouldn't it just be yaml since it's a yaml file? Or is there JSON data embedded in the yaml file?

Reply:
`inspec check` is a linting utility which checks several aspects of a profile, including the inspec.yml file. The `--format` flag controls the output format of `inspec check`. It provides a human-readable ("doc") format, and a machine-readable (json) format.